libs/wallet/src/connect-dialog/view-connector-form.tsx

```diff
@@ -41,24 +49,46 @@ export function ViewConnectorForm({
   }
 
   return (
-    <form onSubmit={handleSubmit(onSubmit)} data-testid="view-connector-form">
-      <FormGroup label={t('Vega Pubkey')} labelFor="address">
-        <Input
-          {...register('address', {
-            required: t('Required'),
-            validate: validatePubkey,
-          })}
-          id="address"
-          data-testid="address"
-          type="text"
-        />
-        {errors.address?.message && (
-          <InputError intent="danger">{errors.address.message}</InputError>
-        )}
-      </FormGroup>
-      <Button data-testid="connect" variant="primary" type="submit" fill={true}>
-        {t('Connect')}
-      </Button>
-    </form>
+    <>
+      <button
+        onClick={reset}
+        className="absolute p-2 top-0 left-0 md:top-2 md:left-2"
+        data-testid="back-button"
+      >
+        <Icon name={'chevron-left'} ariaLabel="back" size={4} />
```

Comment:
suggestion (non-blocking): maybe a text with "Back" could be useful and more obvious for the user then just the small icon here, but this is just my biased take

Reply:
Thanks for the suggestion. I think this is unobtrusive, and if the user is looking for a back button, they'll notice it. It follows the same convention as the close button which is a similarly positioned, similarly sized X. I think the two in conjunction, along with their placement, makes for a nice piece of simple UI.